### PR TITLE
docs: Expand feature toggle type documentation, link from important concepts.

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR is in response to bit of user feedback that we got about being confused about what the feature toggle types were and where to find information on them. It also (partially) addresses the point about it being hard to find when getting started by linking to the info on toggle types in the important concepts chapter.

Ultimately, I think this could do with a bit of restructuring, but this should at least get us a step further 🎉

### CODE REVIEW

File: website/docs/advanced/feature-toggle-types.md [hunk 1]
```diff
@@ -5,24 +5,38 @@ title: Feature Toggle Types
...
+- **Experiment** - Perform multivariate or A/B testing. _Expected lifetime 40 days_
+- **Operational** - Control operational aspects of the system's behavior. _Expected lifetime 7 days_
+- **Kill switch** - Gracefully degrade system functionality. _(permanent)_
+- **Permission** - Change the features or product experience that certain users receive. _(permanent)_
+
+## Deprecating feature toggles {#deprecate-a-feature-toggle}
+
+You can mark feature toggles as `stale`. This is a way to deprecate a feature toggle without removing the active configuration for connected applications. Use this to signal that you should stop using the feature in your applications. Stale toggles will show as stale in the ["technical debt dashboard"](../user_guide/technical_debt).
+
+When you mark a toggle as stale, Unleash will emit an event. You can use [an addon](https://docs.getunleash.io/addons/index) to integrate this with your systems, by for instance posting a message in a Slack channel.
```
Comment (chriswk) [inline website/docs/advanced/feature-toggle-types.md:36]:
```suggestion
When you mark a toggle as stale, Unleash will emit an event. You can use [an addon](https://docs.getunleash.io/addons/index) to integrate this with your systems, for instance to post a message in a Slack channel.
```

File: website/docs/advanced/feature-toggle-types.md [hunk 2]
```diff
@@ -5,24 +5,38 @@ title: Feature Toggle Types
...
+
+Additionally, with some extra work, you can also use the `stale` property to:
+
+- Inform a developers that a toggle is stale _while_ they're developing.
```
Comment (chriswk) [inline website/docs/advanced/feature-toggle-types.md:40]:
```suggestion
- Inform developers that a toggle is stale _while_ they're developing.
```
or
```suggestion
- Inform a developer that a toggle is stale _while_ they're developing.
```

File: website/docs/user_guide/important-concepts.md [hunk 3]
```diff
@@ -31,10 +31,16 @@ Since the SDKs perform local evaluation, some of the parameters needed for evalu
 
 ## API architecture
 
-The Unleash API is split into two. One API is for the clients connecting unleash and is located under the path /api/client, and provides access to retrieving saved feature toggle configurations, metrics and registering the application. 
+The Unleash API is split into two. One API is for the clients connecting unleash and is located under the path /api/client, and provides access to retrieving saved feature toggle configurations, metrics and registering the application.
```
Comment (chriswk) [inline website/docs/user_guide/important-concepts.md:34]:
```suggestion
The Unleash API is split into two. One API is for the clients connecting to unleash. It is located under the path /api/client. This provides access to retrieving saved feature toggle configurations, metrics and registering the application.
```

**Review by chriswk — APPROVED** [review]
Some small nits with spelling/grammar; feel free to disagree.
The text looks good overall, so still going to approve this, but please consider my suggestions.

**Comment by thomasheartman** [comment]
@chriswk Love it! Thanks for catching those; all added 🎉

**Pull request merged**